ext/src/ruby_api/module.rs

```diff
@@ -106,6 +109,24 @@ impl Module {
     pub fn get(&self) -> &ModuleImpl {
         &self.inner
     }
+
+    /// @yard
+    /// Returns the list of imports that this Module has and must be satisfied.
+    /// @return [Array<Hash>] An array of hashes containing import information
+    pub fn imports(&self) -> Result<RArray, Error> {
+        let module = self.get();
+        let imports = module.imports();
+
+        let result = RArray::new();
+        for import in imports {
+            let hash = RHash::new();
+            hash.aset("module", import.module())?;
+            hash.aset("name", import.name())?;
+            hash.aset("type", format!("{:?}", import.ty()))?;
```

Comment:
I'm not sure we want to return a string for the type, ideally we'd want a proper type representing the extern. I believe you can use https://github.com/bytecodealliance/wasmtime-rb/blob/main/ext/src/ruby_api/externals.rs#L110